### ISSUE TITLE
2022-1-30

### ISSUE THREAD

**Issue by ssfc** [ssfc/wangdao#726]
491	Chapter 1, introduction to pytorch; 
492	LEARN THE BASICS; Running the Tutorial Code; How to Use this Guide; (2022-1-21)
493	QUICKSTART; Working with data; torch.utils.data.DataLoader and torch.utils.data.Dataset; (1) Dataset stores the samples and their corresponding labels, (2) DataLoader wraps an iterable around the Dataset; 
494	// the file format is py, not ipynb (short for ipython notebook); (2022-1-21)
495	// we can print the model’s properties; (2022-1-21)
496	// add del model according to leehongyi’s homework 1; (2022-1-21)
497	// use ############# to separate paragraph; (2022-1-21)
498	TENSORS; (2022-1-22)
499	DATASETS & DATALOADERS; Dataset stores the samples and their corresponding labels, and DataLoader wraps an iterable around the Dataset to enable easy access to the samples; (2022-1-22)
